worlds/oot_soh/Items.py

```diff
@@ -28,7 +28,7 @@ class SohItemData(NamedTuple):
 
 
 item_data_table: dict[Items, SohItemData] = {
-    # Items commented out that can never appear in the item pool and are only used on Ship internally
+    # Items commented out that can never appear in the item pool and are only used on Ship internally or Universal Tracker
```

Comment:
```suggestion
    # Items commented out that can never appear in the item pool and are only used on Ship internally
```
You didn't add a commented out item